### PR TITLE
New Functions.

### PR DESCRIPTION
**Pull request by MKM12345**
# What's in this usefulib?
In this usefulib are two new functions:
The is_palindrome() function takes a string as input and checks whether it is a palindrome. A palindrome is a word, phrase, number, or sequence of characters that reads the same forward and backward (ignoring punctuation, and capitalization), such as "racecar" or "A man, a plan, a canal, Panama!". The function first converts the input string to lowercase and treats whitespaces as characters. It then checks whether the resulting string is equal to its reverse, and returns True if it is, or False otherwise.

The calculate_fibonacci() function takes a non-negative integer n as input and calculates the nth number in the Fibonacci sequence using recursion. The Fibonacci sequence is a series of numbers in which each number (starting from the third number) is the sum of the two preceding numbers. For example, the first ten numbers in the Fibonacci sequence are 0, 1, 1, 2, 3, 5, 8, 13, 21, and 34. The calculate_fibonacci() function uses recursion to calculate the nth number in the sequence by recursively calling itself with n-1 and n-2 as inputs until it reaches the base case of n=0 or n=1, at which point it returns the corresponding value in the sequence (0 or 1). The function returns the nth number in the sequence.

## Did you add tests?
Yes, I was able to with the (self).
## Does this solve an issue?
No, just adds 2 functions.

## Checklist
- [✅] Have you followed the [contributing guidelines](https://github.com/hamdivazim/usefulib/blob/main/CONTRIBUTING.md)?
- [✅] Have you added your usefulib to the [list of usefulibs](https://github.com/hamdivazim/usefulib/blob/main/ALLFUNCTIONS.md)?
- [✅] Have you added a test (or explained why you couldn't above) using the guide [here](https://github.com/hamdivazim/usefulib/blob/main/CONTRIBUTING.md#writing-tests)?

### CODE REVIEW

**Comment by MKM12345** [comment]
On behalf of my now-deleted other account: TheCodingLedendofTheNether

**Comment by hamdivazim** [comment]
Ok cool great that you managed to recover it :)

File: usefulib/_usefulibs.py
Comment (hamdivazim) [inline usefulib/_usefulibs.py:1]: @MKM12345 may I ask if there's any reason for removing all the enters in between lines? I noticed this during reviews

File: usefulib/tests.py
Comment (hamdivazim) [inline usefulib/tests.py:1]: Sam here @MKM12345 removal of the enters between lines?

**Review by hamdivazim — COMMENTED** [review]
Looks good @MKM12345, great job 👍

**Comment by MKM12345** [comment]
Sorry - must have been a mistake but you can correct it if you want.

**Comment by hamdivazim** [comment]
@MKM12345 there are merge conflicts and I can't merge. Could you resolve them (syncing the fork should do the job)?

**Comment by MKM12345** [comment]
I cant sync this fork without discarding my commits - could you add them then meantion this pr then

**Pull request closed without merging**